regen/warnings.pl

```diff
@@ -884,6 +884,9 @@ =head1 SYNOPSIS
     use warnings;
     no warnings;
 
+    # On by default for 5.35 and above
```

Comment:
Could be a somewhat confusing comment, since there are warnings always on by default already.

Maybe "Standard warnings enabled by use v5.35 or above"?

Reply:
Done